migrations/sql-migrations/dynamoToSql/Helper.js

```diff
@@ -101,7 +114,7 @@ class Helper {
       sample_file_type: sampleFileTypeEnumKey,
       size: fileSize,
       s3_path: s3Path,
-      upload_status: file.upload.status,
+      upload_status: file.upload?.status || "uploaded",
```

Comment:
If upload status is malformed we want to say the sample file is uploaded anyways? I trhink it's better if it fails and to check why it failed